.github/workflows/issue-label-assign.yml

```diff
@@ -83,24 +90,35 @@ jobs:
            {"keywords":["(@aws-cdk/aws-ec2)","(aws-ec2)","(ec2)","(vpc)"],"labels":["@aws-cdk/aws-ec2"],"assignees":["rix0rrr"]},
            {"keywords":["(@aws-cdk/aws-ecr)","(aws-ecr)","(ecr)"],"labels":["@aws-cdk/aws-ecr"],"assignees":["MrArnoldPalmer"]},
            {"keywords":["(@aws-cdk/aws-ecr-assets)","(aws-ecr-assets)","(ecr-assets)","(ecr assets)","(ecrassets)"],"labels":["@aws-cdk/aws-ecr-assets"],"assignees":["eladb"]},
+           {"keywords":["(@aws-cdk/aws-ecs)","(aws-ecs)","(ecs)"],"labels":["@aws-cdk/aws-ecs"],"assignees":["MrArnoldPalmer"]},
```

Comment:
At the least, ecs and ecs-patterns should also go to uttarasridhar and SoManyHs. However, they also want these in their project board. I haven't included these in the yaml because they've asked me to add these to their project boards, and if I don't see them come in (by looking at unlabelled issues) then I can't do that for them because our automation currently doesn't allow for us to automatically add to project boards (We have a goal to implement this by end of Q2)

@SoManyHs, what would you prefer here? Do you mind adding these to the project board yourself if these issues get automatically assigned to you?